manifests/i/Inkdrop/Inkdrop/5.8.1/Inkdrop.Inkdrop.locale.en-US.yaml

```diff
@@ -0,0 +1,139 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Inkdrop.Inkdrop
+PackageVersion: 5.8.1
+PackageLocale: en-US
+Publisher: Inkdrop

```

Comment:
```suggestion
Publisher: Takuya Matsuyama
```
This should match what is written to control panel
![image](https://github.com/microsoft/winget-pkgs/assets/12611259/54923244-38d0-4e89-b969-ca674b86109e)